models/census.js

```diff
@@ -3,22 +3,14 @@
 var mongoose = require('mongoose');
 
 var censusSchema = mongoose.Schema({
-    'type': String,
     'activity_year': String,
-    'code': String,
-    'name': String,
+    'msa_code': String,
+    'msa_name': String,
+    'state_code': String,
+    'county_code': String,
     'small_county': String,
-    'state': [{
-        'fips_code': String,
-        'name': String
-    }],
-    'county': [{
-        'fips_code': String,
-        'name': String,
-        'small_county': String
-    }],
-    'tract': [String]
+    'tract': String
 });
-censusSchema.index({'activity_year': 1, 'code': 1, 'type': 1, 'tract': 1});
-censusSchema.index({'activity_year': 1, 'code': 1, 'type': 1, 'small_county': 1});
+censusSchema.index({'activity_year': 1, 'code': 1, 'tract': 1});
+censusSchema.index({'activity_year': 1, 'code': 1, 'small_county': 1});
```

Comment:
These indexes need updated more than just removing `type`, since `code` doesn't exist either.
Based on the queries done, these need  to be updated/replaced with the following:
```
censusSchema.index({'activity_year': 1, 'msa_code': 1, 'state_code': 1, 'county_code': 1, 'tract': 1});
censusSchema.index({'activity_year': 1, 'state_code': 1, 'county_code': 1, 'tract': 1});
censusSchema.index({'activity_year': 1, 'state_code': 1, 'county_code': 1, 'small_county': 1});
```